src/clj/clj_diff/miller.clj

```diff
@@ -218,18 +224,22 @@
   (map #(vec (cons ::sentinel %)) more))
 
 (defn order->ses
-  [a b opts]
-  (let [[a* b*] (if (> (count b) (count a)) [b a] [a b])]
-    [(ses a* b* opts) a* b*]))
+  ([a b]
+   (order->ses a b))
+  ([a b opts]
+   (let [[a* b*] (if (> (count b) (count a)) [b a] [a b])]
+     [(ses a* b* opts) a* b*])))
 
 (defn seq-diff
-  [opts a b]
-  (let [[a b] (vectorize a b)
-        [es a* b*] (order->ses a b opts)
-        edits (apply edits opts a* b* es)]
-    (edits->script b edits (if (= a* a) identity transpose))))
-
-(defn string-dispatch [a b & [_]]
+  ([a b]
+   (seq-diff a b {}))
+  ([a b opts]
+   (let [[a b] (vectorize a b)
+         [es a* b*] (order->ses a b opts)
+         edits (apply edits opts a* b* es)]
+     (edits->script b edits (if (= a* a) identity transpose)))))
+
+(defn string-dispatch [a b & _]
```

Comment:
It should just be ignoring `opts` as its no longer variadic.
```suggestion
(defn string-dispatch [a b _]
```

If you're doing this to account for 2- or 3-arity, please use the `([a b] (string-dispatch {})) ([a b _] ...)` convention we've been using instead.